src/ServiceInsight/Framework/Modules/CoreModule.cs

```diff
@@ -21,6 +21,7 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<ServiceControlConnectionProvider>().InstancePerLifetimeScope();
             builder.RegisterType<DefaultServiceControl>().As<IServiceControl>().InstancePerLifetimeScope();
             builder.RegisterType<CommandLineArgParser>().SingleInstance().OnActivating(e => e.Instance.Parse());
+            builder.RegisterType<WorkNotifier>().As<IWorkNotifier>();
```

Comment:
`WorkNotifier` is passed an `IEventAggregator` in its constructor and I see [`IEventAggregator` being registered with `InstancePerLifetimeScope`](https://github.com/Particular/ServiceInsight/blob/c52d649923798f2472a529fb37b57bd9f1c552a2/src/ServiceInsight/Framework/Modules/CaliburnModule.cs#L16).

If I understand correctly then the registration of `WorkNotifier` cannot be `SingleInstance`, but I guess it probably makes sense to increase its lifetime to `InstancePerLifetimeScope`.